srcpkgs/sshguard/files/sshguard/run

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Allow the firewall and logger backends to be specified
+[ -f ./conf ] && . ./conf
+
+sv check "${FIREWALL:-iptables}" >/dev/null || exit 1
+
+sv check "${LOGGER:-socklog-unix}" >/dev/null || exit 1
```

Comment:
Is there actually a reason to check for that? I would guess sshguard to use inotify or reopen the file and try to read after the last offset, I don't see why it should have to check for socklog unix to actually be running.